doc/release/release_0.16.rst

```diff
@@ -0,0 +1,384 @@
+- `skimage.transform.seam_carve` has been completely removed from the
+  library due to licensing restrictions.
+- Parameter ``as_grey`` has been removed from `skimage.data.load` and
+  `skimage.io.imread`. Use ``as_gray`` instead.
+- Parameter ``min_size`` has been removed from
+  `skimage.morphology.remove_small_holes`. Use ``area_threshold`` instead.
+- Deprecated ``correct_mesh_orientation`` in `skimage.measure` has been
+  removed.
+- `skimage.measure._regionprops` has been completely switched to using
+  row-column coordinates. Old x-y interface is not longer available.
+- Default value of ``behavior`` parameter has been set to ``ndimage`` in
+  `skimage.filters.median`.
+- Parameter ``flatten`` in `skimage.io.imread` has been removed in
+  favor of ``as_gray``.
+- Parameters ``Hxx, Hxy, Hyy`` have been removed from
+  `skimage.feature.corner.hessian_matrix_eigvals` in favor of ``H_elems``.
+- Default value of ``order`` parameter has been set to ``rc`` in
+  `skimage.feature.hessian_matrix`.
+- ``skimage.util.img_as_*`` functions no longer raise precision and/or loss warnings.
+
+Bugfixes
+--------
+
+- Corrected error with scales attribute in ORB.detect_and_extract (#2835)
+  The scales attribute wasn't taking into account the mask, and thus was using
+  an incorrect array size.
+- Correct for bias in Inverse Randon Transform (`skimage.transform.irandon`) (#3067)
+  Fixed by using the Ramp filter equation in the spatial domain as described
+  in the reference
+- Fix a rounding issue that caused  a rotated image to have a
+  different size than the input (`skimage.transform.rotate`)  (#3173)
+- RANSAC uses random subsets of the original data and not bootstraps. (#3901,
+  #3915)
+- Canny now produces the same output regardless of dtype (#3919)
+- Geometry Transforms: avoid division by zero & some degenerate cases (#3926)
+- Fixed float32 support in denoise_bilateral and denoise_tv_bregman (#3936)
+- Fixed computation of Meijering filter and avoid ZeroDivisionError (#3957)
+- Fixed `skimage.filters.threshold_li` to prevent being stuck on stationnary
+  points, and thus at local minima or maxima (#3966)
+- Edited `skimage.exposure.rescale_intensity` to return input image instead of
+  nans when all 0 (#4015)
+- Fixed `skimage.morphology.medial_axis`. A wrong indentation in Cython
+  caused the function to not behave as intended. (#4060)
+- Fixed `skimage.restoration.denoise_bilateral` by correcting the padding in
+  the gaussian filter(#4080)
+- Fixed `skimage.measure.find_contours` when input image contains NaN.
+  Contours interesting NaN will be left open (#4150)
+- Fixed `skimage.feature.blob_log` and `skimage.feature.blob_dog` for 3D
+  images and anisotropic data (#4162)
+- Fixed `skimage.exposure.adjust_gamma`, `skimage.exposure.adjust_log`,
+  and `skimage.exposure.adjust_sigmoid` such that when provided with a 1 by
+  1 ndarray, it returns 1 by 1 ndarrays and not single number floats (#4169)
```

Comment:
still hoping we can merge #3584 and add it here as well (appropriate text for the notes is included in that PR's `release_dev.txt`)

Reply:
I tagged #3584 as release critical.

Reply:
Merged #3584.